container_usage.md

```diff
@@ -1,5 +1,79 @@
 # 🐳 Running Jikan API in a container
 
+The most easiest way to get started is to use our container setup cli script after checking out the repo with git (linux only):
+
+```bash
+./container-setup.sh start
+```
+
+This will:
+
+- Prompt you for the required passwords and usernames
+- Sets up a production ready setup with `redis`, `typesense` and `mongodb` (almost same as the public api at `api.jikan.moe`)
+- Sets mongodb to use max 1gb of memory
+- Configures jikan-api to add CORS headers to responses.
+
+> **Note**: The script supports both `docker` and `podman`. In case of `podman` please bare in mind that sometimes the container name resolution doesn't work on the container network. 
+> In those cases you might have to install `aardvark-dns` package. On `Arch Linux` podman uses `netavark` network by default (in 2023) so you will need to install the before mentioned package.
+
+The script has the following prerequisites and will notify you if these are not present:
+
+- git
+- `docker` or `podman`
+- `docker-compose` or `podman-compose`
+
+### Available commands in the cli script
+
+```
+============================================================
+Jikan API Container Setup CLI
+============================================================
+Syntax: ./container-setup.sh [command]
+---commands---
+help                   Print CLI help
+build-image            Build Image Locally
+start                  Start Jikan API (mongodb, typesense, redis, jikan-api workers)
+stop                   Stop Jikan API
+validate-prereqs       Validate pre-reqs installed (docker, docker-compose)
+execute-indexers       Execute the indexers, which will scrape and index data from MAL. (Notice: This can take days)
+```
+
+### Running the indexer with the script
+
+When you first startup the app you will have an empty database. To fill it up you can execute the following command:
+
+```bash
+./container-setup.sh execute-indexers
+```
+
+Please note that this command can take 4-5 days to run. You can run it in the background with the `&` marker:
+
+```bash
+./container-setup.sh execute-indexers &
+```
+
+If interrupted then you will have to manually resume the indexing, otherwise the above command will just start again from the beginning.
+
+### Updating to a newer version
+
+You need to stop the app first:
+
+```bash
+./container-setup.sh stop
+```
+
+Then remove the jikan-api image from your local storage and pull the new one. Set the `JIKAN_API_VERSION` environment variable to the latest image tag. This can be either `latest` or the version `v4.0.0-11`.
```

Comment:
What is this version `v4.0.0-11` corresponding to?

Reply:
That's the tag name for the container image. But that's just a guessed format from me. In the registry the tag names will always match the release name. So each time you run the github action to release an image to the container registry, the built image will have the tag name from the release, or the tag name you specify when you start it manually, and additionally it will have the "latest" tag if the workflow was  triggered by a release. "latest-nightly" if you have triggered it manually.